src/SumoCoders/FrameworkCoreBundle/Service/KeyValueStore.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Service;
+
+/**
+ * Class KeyValueStore
+ *
+ * @package SumoCoders\FrameworkCoreBundle\Service
+ */
+class KeyValueStore
```

Comment:
Also, if you do create your own custom key/value store, it would be better to just provide an interface `KeyValueStore`, and make Fallbacks & JsData classes implement it. These classes are very specific, and right now they extend the `KeyValueStore` class while adding new methods, which violates the Liskov Substitution Principle...